src/core/hotkeys/linux/x11.cpp

```diff
@@ -119,7 +119,10 @@ namespace Soundux::Objects
     void Hotkeys::stop()
     {
         kill = true;
-        listener.join();
+        if (listener.joinable())
```

Comment:
Why would you check for this?
The thread will always be joinable as long as you consistently call start/stop